### PR TITLE
#166571933 Fix app responsiveness and social login styling

### PR DESCRIPTION
**Pull request by jkamz**
### What does this PR do?
Fix app responsiveness and stlye social buttons

### Description of Task(s) to be completed?
- Add colors to variables.css
- Fix styling of social login buttons on some browsers
- Fix responsiveness of signup page
- Fix responsiveness of login page

### How should this be manually tested?

Clone this repository on local machine and CD into project folder
`$ git clone git@github.com:andela/ah-the-jedi-frontend.git`
`$ cd ah-the-jedi-frontend`

Check out to the PR's branch
`$ git checkout bg-fix-app-responsiveness-166571933`

install dependencies and run the dev server
`npm install`
`npm run start:dev`

navigate to login and signup page and change sizes to view responsiveness

### What are the relevant PT stories?
[#166571933](https://www.pivotaltracker.com/story/show/166571933)

#### Sign Up
image

#### Sign In
image

### CODE REVIEW

File: tests/Social.test.js [hunk 1]
```diff
@@ -108,7 +108,6 @@ describe('unit tests for clicking social buttons', () => {
   it('should call method', () => {
     const component = shallow(<FacebookLogin />);
     expect(component.exists()).toEqual(true);
-    console.log(tree.props());
```
Comment (Elisha-Misoi) [inline tests/Social.test.js]: Please remove this line.

**Comment by katherine95** [comment]
Great work on implementing app responsiveness on sign up and log in. What about on the navigation bar after a user has logged in? That is onClicking the drop down.
Screenshot 2019-06-11 at 13 13 35

**Comment by jkamz** [comment]
> Great work on implementing app responsiveness on sign up and log in. What about on the navigation bar after a user has logged in? That is onClicking the drop down.
> Screenshot 2019-06-11 at 13 13 35
Hey @katherine95 . Thanks for your suggestion. I have fixed it please check.

**Review by ElishaMisoi — APPROVED** [review]

**Review by Siffersari — APPROVED** [review]
Good job fixing the responsiveness. Great implementation for the Navbar responsiveness also. @jkamz 

responsive

**Comment by katherine95** [comment]
> Good job fixing the responsiveness. Great implementation for the Navbar responsiveness also. @jkamz
> 
> responsive

great job implementing feedback.

**Review by katherine95 — APPROVED** [review]

**Review by AnguleMathias — APPROVED** [review]

**Pull request merged**